tensorflow/core/kernels/eigen_spatial_convolutions-inl.h

```diff
@@ -347,13 +378,131 @@ class TensorContractionInputMapper<
     if (nonStandardPatches()) {
       return packetWithPossibleZero(patchId, rowIndex, colIndex, otherIndex);
     }
-    return loadPacketStandard(patchId, rowIndex, colIndex, otherIndex);
+    typedef decltype(m_impl) TensorEvaluatorT;
+    return loadPacketStandard<Packet, TensorEvaluatorT>(patchId, rowIndex,
+                                                        colIndex, otherIndex);
+  }
+
+  // Helper function to load a 'partial' packet - this is the single column
+  // part of a packet that is split across two columns.
+  // If required, this function is called from
+  // loadPacketStandardFromTwoColumns() when the packet type supports masked
+  // load and when the partial packet load is available in the TensorEvaluator.
+  EIGEN_DEVICE_FUNC
+  EIGEN_ALWAYS_INLINE Packet loadPartialPacketStandard(
+      Index patchId, Index rowIndex, Index colIndex, Index otherIndex,
```

Comment:
You can avoid passing `pathId` and `patchIdPart` if you modify the coefficient loop at the end to something like

```
for (int I = 0; i < spanPart[1] - spanPart[0] + 1; ++i)
	        values[i] = loadCoeff(patchIdPart + i, rowIndex, colIndex, otherIndex);
```

I'd prefer to get rid of if `?Part` suffix in all variable names, just a patchId (within patch offset) + span.